### PR TITLE
inline-model-metrics

### PR DESCRIPTION
**Pull request by jonseddon**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jonseddon** [comment]
I consent to being listed here

**Comment by robertsmalcolm** [comment]
I consent to being listed here

File: recipes/inline-model-metrics/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/inline_model_metrics-{{ version }}.tar.gz
+  sha256: 0f0ffef1ae2d5450698f926e79124a49de2edc540a62122d6b59754d05fa5fb8
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: True  # [win or osx]
```
Comment (dopplershift) [inline recipes/inline-model-metrics/meta.yaml:15]: This looks like it's pure Python:
```suggestion
  noarch: python
```

we can ignore the resulting CI failures from `metoffice-afterburner` only being available on Linux (noarch packages are only normally built on Linux).

**Comment by jonseddon** [comment]
@dopplershift I don't suppose that you can help please? The Linux builds are failing and I can't spot the source of the error. There is a [line that says](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=628723&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=1348):

```
ERROR: [umfilelist.<module>:254] Unable to find 'moo' command in runtime environment
```
but this is just a logger message to stderr from afterburner.contrib.umfilelist.py and shouldn't be raising an error.

Later in the logs[ it says ](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=628723&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=1351)and I assume that this is the actual error:

```
+ pip check
metoffice-afterburner 1.3.2b1.post4 has requirement scitools-iris<3,>=2.1, but you have scitools-iris 3.4.0.
Tests failed for inline-model-metrics-0.1.2-pyhd8ed1ab_0.conda - moving package to /home/conda/staged-recipes/build_artifacts/broken
WARNING:conda_build.build:Tests failed for inline-model-metrics-0.1.2-pyhd8ed1ab_0.conda - moving package to /home/conda/staged-recipes/build_artifacts/broken
```

but I'm really struggling to see the cause of both the scitools-iris (I've changed in meta.yml from scitools-iris to iris) and the <3. [metoffice-afterburner's meta.yml](https://github.com/conda-forge/metoffice-afterburner-feedstock/blob/main/recipe/meta.yaml) asks for `iris >=2.4` but has no upper limit on scitools-iris. I can't see limits in other packages. 

On my desktop then `conda create -n afterburner -c conda-forge "python>=3.6,<3.10"  metoffice-afterburner scitools-iris netcdf4 numpy tempest-helper` should be the same as inline-model-metric's meta.yml and it installs all of these packages, including scitools-iris 3.4 without any issues.

Thanks for any pointers!

File: recipes/inline-model-metrics/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    - numpy
+    - tempest-helper
+
+test:
+  imports:
+    - inline_model_metrics
+  commands:
+    - pip check
+  requires:
+    - pip
```
Comment (dopplershift) [inline recipes/inline-model-metrics/meta.yaml:35]: You can remove the call to `pip check`. This is unlikely ever to work due to the difference in naming of `iris` vs. `scitools-iris` for conda-forge vs. PyPI. The import check itself seems pretty robust for this package to make sure it's properly working and has all dependencies installed.

```suggestion
```

**Review by dopplershift — APPROVED** [review]
Ignoring noarch Windows and macOS build failures due to missing dependency.

**Comment by jonseddon** [comment]
Thanks for you help and advice @dopplershift!

**Pull request merged**